src/box/lua/load_cfg.lua

```diff
@@ -1209,6 +1222,28 @@ local function get_option_from_env(option)
     if param_type:find('table') and (raw_value:startswith('{') or
                                      raw_value:startswith('[')) then
         return json.decode(raw_value)
+    elseif param_type:find('uri_list') then
```

Comment:
Instead of introducing the new pseudo-type `uri_list`, what about *not* interpreting a string like `k1=v1,k2=v2,...` as a map in case `ki` contains `?`? I doubt anyone would use `?`.

Reply:
Good idea! I've implemented it. Please, take a look.